.github/workflows/downstream_protobuf_compatibility_check_nightly.yaml

```diff
@@ -47,8 +47,13 @@ jobs:
         uses: actions/checkout@v4
       - uses: actions/setup-java@v4
         with:
-          # Use Java 11 for this as Linkage Checker is only compatible with Java 11 or below
-          java-version: 11
+          # TODO(https://github.com/GoogleCloudPlatform/cloud-opensource-java/issues/2395): Java 17+ support for Linkage Checker
+          java-version: 8
+          distribution: temurin
+      - run: echo "JAVA8_HOME=${JAVA_HOME}" >> $GITHUB_ENV
+      - uses: actions/setup-java@v4
+        with:
+          java-version: 17
```

Comment:
Does this (using Java 17 to compile code) catch the Protobuf 4.30.1 Java 8 incompatibility issue?

Reply:
Turns out it does not. Seems like it resolves the class file issue but wouldn't be able to test for the Protobuf v30.x issue.